resources/js/fetch.js

```diff
@@ -37,6 +37,8 @@ window.rapidezAPI = async (method, endpoint, data = {}, options = {}) => {
     let response = await rapidezFetch(window.url('/api/' + endpoint), {
         method: method.toUpperCase(),
         headers: {
+            Store: window.config.store_code,
+            Authorization: token.value ? `Bearer ${token.value}` : null,
```

Comment:
Can this still be overwritten using the options parameter?
In some cases we still need to send the cart hash via the authorization header right?